package.json

```diff
@@ -2,17 +2,11 @@
   "name": "@altinn/altinn-components",
   "version": "0.18.1",
   "main": "dist/index.js",
-  "files": [
```

Comment:
We need to align our formatting, we keep reformatting back and forth 😆

Reply:
Agreed! The linting is going rogue 😆